pywebchannel/qwebchannel.py

```diff
@@ -31,6 +31,9 @@ class QWebChannel(object):
     def __init__(self, initCallback=None):
         self.initCallback = initCallback
         self.__initialized = False
+        self.objects = {}
+        self.execCallbacks = {};
+        self.execId = 0;
```

Comment:
No need for semicolons:
```suggestion
        self.execCallbacks = {}
        self.execId = 0
```